src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/CoreInstanceMetaData.java

```diff
@@ -8,21 +8,18 @@
     private Integer volumeCount;
     private String longName;
     private Integer containerCount = 0;
+    private String hostGroup;
```

Comment:
We should be consequent with the naming conventions - hostGroups are for the blueprint/cluster layer and instanceGroup is for the stack/instance layer.